torchvision/models/detection/_utils.py

```diff
@@ -527,7 +527,12 @@ def _box_loss(
         target_regression = box_coder.encode_single(matched_gt_boxes_per_image, anchors_per_image)
         beta = cnf["beta"] if cnf is not None and "beta" in cnf else 1.0
         return F.smooth_l1_loss(bbox_regression_per_image, target_regression, reduction="sum", beta=beta)
-    else:  # giou
+    else:
         bbox_per_image = box_coder.decode_single(bbox_regression_per_image, anchors_per_image)
         eps = cnf["eps"] if cnf is not None and "eps" in cnf else 1e-7
-        return generalized_box_iou_loss(bbox_per_image, matched_gt_boxes_per_image, reduction="sum", eps=eps)
+        if type == "ciou":
+            return complete_box_iou_loss(bbox_per_image, matched_gt_boxes_per_image, reduction="sum", eps=eps)
+        elif type == "diou":
```

Comment:
Hi @datumbox , Just a small code style questions here, do you think it will be better to use the following style?

```python
        if type == "ciou":
            return complete_box_iou_loss(bbox_per_image, matched_gt_boxes_per_image, reduction="sum", eps=eps)
        if type == "diou":
            return distance_box_iou_loss(bbox_per_image, matched_gt_boxes_per_image, reduction="sum", eps=eps)
        # Otherwise giou
        return generalized_box_iou_loss(bbox_per_image, matched_gt_boxes_per_image, reduction="sum", eps=eps)
```

Reply:
I'm OK with this. Thanks!